docs/platforms/javascript/common/opentelemetry/custom-setup.mdx

```diff
@@ -95,13 +108,24 @@ By default, Sentry will register OpenTelemetry instrumentation to automatically
 
 If tracing is not enabled (no `tracesSampleRate` is defined in the SDK configuration), only a minimal amount of OpenTelemetry instrumentation will be registered. This includes the following:
 
+{/* prettier-ignore-start */}
+
 - <PlatformLink to="/configuration/integrations/http/">httpIntegration</PlatformLink> registers [@opentelemetry/instrumentation-http](https://www.npmjs.com/package/@opentelemetry/instrumentation-http)
 - <PlatformLink to="/configuration/integrations/nodefetch/">nativeNodeFetchIntegration</PlatformLink> registers [opentelemetry-instrumentation-fetch-node](https://www.npmjs.com/package/opentelemetry-instrumentation-fetch-node)
 
-<PlatformSection notSupported={['javascript.aws-lambda', 'javascript.gcp-functions']}>
+{/* prettier-ignore-end */}
+
+<PlatformSection
+  notSupported={["javascript.aws-lambda", "javascript.gcp-functions"]}
+>
   <Note>
-    If tracing is not enabled, performance instrumentations will not be registered but they will still be included in the bundle. If you
-    want to reduce the bundle size or used dependencies, you can also <PlatformLink to="/configuration/tree-shaking/#setting-up-sentry-without-performance-integrations">Set up Sentry without Performance Integrations</PlatformLink>.
+    If tracing is not enabled, performance instrumentations will not be
+    registered but they will still be included in the bundle. If you want to
+    reduce the bundle size or used dependencies, you can also{" "}
```

Comment:
```suggestion
    reduce the bundle size or used dependencies, you can also
```

Reply:
These are necessary and auto inserted by our formatter.